templates/ampache.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0"?>
+<Container version="2">
+  <Name>ampache</Name>
+  <Repository>ampache/ampache</Repository>
+  <Registry>https://hub.docker.com/r/ampache/ampache/</Registry>
+  <Network>bridge</Network>
+  <Shell>sh</Shell>
+  <Privileged>false</Privileged>
+  <Support>https://github.com/ampache/ampache/issues</Support>
+  <Project>https://ampache.org/</Project>
+  <Overview>Ampache ...for the love of music</Overview>
+  <WebUI>http://[IP]:[PORT:8084]</WebUI>
```

Comment:
where is this port?

Reply:
added the port now.

Reply:
Port does not match.

Reply:
port patches the default for the WebUI Port. Template is working as is. when i change the port to match the target below, it breaks this functionality.

Reply:
It does not. You have done this error twice. #464, #463

Reply:
my mistake. i thought it was tied to the host port. I saw the message from squid. have since updated.